packages/build/src/log/messages/compatibility.ts

```diff
@@ -161,16 +160,7 @@ export const logIncompatiblePlugins = function (logs, pluginsOptions) {
 }
 
 const hasIncompatibleVersion = function ({ pluginPackageJson: { version }, compatibleVersion, compatWarning }) {
-  return (
-    compatWarning !== '' &&
-    version !== undefined &&
-    compatibleVersion !== undefined &&
-    // Using only the major version prevents printing this warning message when
```

Comment:
Is this no longer a problem?

Reply:
Sadly this code already dates back my knowledge, but based on the comment I would assume that this "race" condition is super rare.

It might occur still but the worst case in this scenario is that we show a misleading warning in those edge cases.

Where as in the case of the next.js runtime we don't show this warnings as all (as it's a newer major by design).

I think for all new major versions that we release on plugins we still want to show why it did not used the incompatible (new major) version?